src/content.js

```diff
@@ -1,29 +1,22 @@
-import { IS_FIREFOX, isSupportExecutionVersion } from './utils'
+import { IS_CHROME, isSupportExecutionVersion } from './utils'
 const browser = require('webextension-polyfill')
 
-// injecting the script, inspire by react-devtools-extensions
-function injectScriptSync (src) {
-  let code = ''
-  const request = new XMLHttpRequest()
-  request.addEventListener('load', function () {
-    code = this.responseText
-  })
-  request.open('GET', src, false)
-  request.send()
+// injecting the script
+function injectScript (src) {
   const script = document.createElement('script')
-  script.textContent = code
-  // This script runs before the <head> element is created, so we add the script to <html> instead.
+  script.setAttribute('defer', 'defer')
```

Comment:
I changed back to the previous injection logic. Because extensions can't set textContent or innerHTML of a script element, it violates the following Content Security Policy directive.